code/controllers/subsystem/mapping.dm

```diff
@@ -167,9 +167,11 @@ SUBSYSTEM_DEF(mapping)
 #define INIT_ANNOUNCE(X) to_chat(world, "<span class='boldannounce'>[X]</span>"); log_world(X)
 /datum/controller/subsystem/mapping/proc/LoadGroup(list/errorList, name, path, files, list/traits, list/default_traits, silent = FALSE)
 	var/start_time = REALTIMEOFDAY
-
 	if (!islist(files))  // handle single-level maps
 		files = list(files)
+	else
+		files = files
```

Comment:
I will remove that tomorrow or Sunday in a cleanup of my old debug stuff that I need to remove.